<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogConflictExpert</class>
 <widget class="QDialog" name="DialogConflictExpert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration conflict - Expert mode</string>
  </property>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>140</y>
     <width>100</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Cancel driver
configuration</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMoreInfo">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>100</width>
     <height>26</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Show values</string>
   </property>
   <property name="text">
    <string>More info ...</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>The IcePAP database differs from the current configuration in the driver board.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>411</width>
     <height>48</height>
    </rect>
   </property>
   <property name="text">
    <string>An expert has probably configured the driver with an external tool.

The current values in the database will be overwritten.</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnUpdate">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>140</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Update database with current
driver board configuration</string>
   </property>
   <property name="text">
    <string>Update database</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
